apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.py

```diff
@@ -115,6 +120,7 @@ def render(element_html: str, data: pl.QuestionData) -> str:
             "markdown_shortcuts": "true" if markdown_shortcuts else "false",
             "counter": counter.value,
             "counter_enabled": counter != Counter.NONE,
+            "clipboard_enabled": "true" if clipboard_enabled else "false",
```

Comment:
Extremely minor nit: You only pass the boolean value of `clipboard_enabled` and then have `"true"` and `"false"` baked into the template. This makes the Python file slightly cleaner and reduces coupling, but won't block on this.

Reply:
Done in 111d33f7ee1a927c13b73c9bd0d112c3b2ed0097.